lib/Sema/TypeCheckAvailability.cpp

```diff
@@ -1499,6 +1499,27 @@ bool TypeChecker::isInsideUnavailableDeclaration(
                                   IsUnavailable);
 }
 
+bool TypeChecker::isInsideCompatibleUnavailableDeclaration(
+	SourceRange ReferenceRange, const DeclContext *ReferenceDC,
+	const AvailableAttr *attr)
+{
```

Comment:
Can you be sure to run this through clang-format? This is not LLVM style. If you use git-clang-format, everything will just work for you.

Reply:
I don't see any mention of formatting in [Contributing Code](https://swift.org/contributing/#contributing-code). Is there another page of documentation that covers this (and possibly other conventions I've missed)?

Reply:
Ah, or maybe Swift just adopts all the conventions from clang... =)

Reply:
That's it, or at least 90% of them, but we should definitely mention that on the page.